.gitignore

```diff
@@ -51,3 +51,6 @@ html/
 
 # VS Code directory
 .vscode
+
+# install directory
+install/
```

Comment:
Btw, you can also add those directory on a per-user config, either for all your git repos e.g.
```
git config --global core.excludesfile $HOME/.gitignore
```
or per-repo e.g.
```
git config core.excludesfile .git/.user_gitignore
```

Reply:
Cool, thx.

Reply:
There's also:

* `$XDG_CONFIG_HOME/git/ignore` (`~/.config/git/ignore` on most systems) for per-user
* `.git/info/exclude` for per-repo

Reply:
Is `install/` _the_ preferred prefix for installations?
In `FairRoot_build_test.cmake` we use `build/install/` for installing.

Reply:
> Is install/ the preferred prefix for installations?

No. That is completely user defined.

One could argue this does not belong here. but we also have `build/` in it, which then consequently also shouldn't be there I guess.

Reply:
I guess we put things in here in the past due to the lack of knowledge about how to add it on a per-user config level.

Reply:
> One could argue this does not belong here. but we also have `build/` in it, which then consequently also shouldn't be there I guess.

For `build/` one could argue, that our CI tools use that. So those are not chosen by the developer.

Reply:
Proposal: Let's add something like this at the top of the `.gitignore` and leave the rest as is for now?

```
#
# Ignore patterns specific to this project
#
# If you need personal additions, consider
# - `$XDG_CONFIG_HOME/git/ignore` (`~/.config/git/ignore` on most systems)
#   for per-user patterns
# - `.git/info/exclude` for this local repository
# Also see "man gitignore", and `core.excludesFile` in "man git-config"
#
```